src/screens/main/main/StoryScreen.tsx

```diff
@@ -0,0 +1,182 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import React from 'react';
+import { StyleSheet, Text, View, Image } from 'react-native';
+import { Screens } from '@navigation/constants';
+import { MainStackParams } from '@navigation/stacks/MainStack';
+import sizes from '@styles/sizes';
+import colors from '@styles/colors';
+import constants from '@utils/constants';
+import Box from '@components/atoms/Box';
+
+const StoryScreen: React.FC<NativeStackScreenProps<MainStackParams, typeof Screens.Main.MAIN>> = props => {
+    const { navigation } = props;
+    return (
+        <View style={styles.wrapper}>
+            <View style={styles.wrapper__content}>
+                <View style={styles.content__profile_wrapper}>
+                    <Image source={require('@assets/icons/profile.png')} style={styles.content__profile} />
+                </View>
+                <View style={styles.content__header}>
+                    <Text style={styles.content__header}>{constants.headerMain}</Text>
+                </View>
+                <View style={styles.content__cover_boxes}>
+                    <Image source={require('@assets/icons/bookCover.png')} />
+                    <View style={styles.wrapper__content_boxes}>
+                        <Box
+                            textHeader={constants.textRating}
+                            textInfo={constants.scoreRating}
+                            imageSource={require('@assets/icons/star.png')}
+                        />
+                        <Box
+                            textHeader={constants.textDuration}
+                            textInfo={constants.duration}
+                            imageSource={require('@assets/icons/clock.png')}
+                        />
+                        <Box
+                            textHeader={constants.textComplete}
+                            textInfo={constants.completePercent}
+                            imageSource={require('@assets/icons/book.png')}
+                        />
+                    </View>
+                </View>
+                <View style={styles.wrapper__content_genres}>
+                    <View style={styles.content__boxg}>
+                        <Text style={styles.content__boxg1}>{constants.textMystic}</Text>
+                    </View>
+                    <View style={styles.content__boxg}>
+                        <Text style={styles.content__boxg2}>{constants.textThriller}</Text>
+                    </View>
+                    <View style={styles.content__boxg}>
+                        <Text style={styles.content__boxg3}>{constants.textRomance}</Text>
+                    </View>
+                </View>
+            </View>
+        </View>
+    );
+};
+
+const styles = StyleSheet.create({
+    wrapper: {
+        flex: 1,
+        flexDirection: 'column',
+        alignItems: 'flex-start',
+        justifyContent: 'space-between',
+        paddingTop: 76,
```

Comment:
This padding may look too big on small devices, it is better to do it with referencing height of header/statusbar etc.